### ISSUE TITLE
pulp_pull plugin not disabled if pulp registry not used

### ISSUE THREAD

**Issue by lcarva** [projectatomic/osbs-client#480]
When using osbs-client without a pulp registry, the `pulp_pull` plugin is not disabled, which causes the build to fail due to assertion error: 

```
2016-11-28 11:39:03,359 - atomic_reactor.plugin - DEBUG - Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugin.py", line 225, in run
    plugin_response = plugin_instance.run()
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugins/post_pulp_pull.py", line 54, in run
    assert self.workflow.push_conf.pulp_registries  # must be configured
AssertionError
2016-11-28 11:39:03,359 - atomic_reactor.plugin - ERROR - plugin 'pulp_pull' raised an exception: AssertionError()
```

**Comment by lcarva** [comment]
Workaround is to manually remove `pulp_pull` plugin from prod_inner.json.

**Comment by lcarva** [comment]
Fixed by #554
